src/modules/sub-items/components/view-switcher/view.switcher.component.js

```diff
@@ -79,5 +85,17 @@ export default class ViewSwitcherComponent extends Component {
 ViewSwitcherComponent.propTypes = {
     onViewChange: PropTypes.func.isRequired,
     activeView: PropTypes.string.isRequired,
-    isDisabled: PropTypes.bool.isRequired
+    isDisabled: PropTypes.bool.isRequired,
+    labels: PropTypes.shape({
+        title: PropTypes.object.isRequired
```

Comment:
The `labels` object should be as flat as possible, so `listViewTitle` and `gridViewTitle` are ok. No need to nest the `title` sub-object.